### PR TITLE
Fixed Sound Error Bug

### PR DESCRIPTION
**Pull request by TheRiddler4K**
All sounds have been moved to InGameView which fixes our problem

### CODE REVIEW

File: my_game.py [hunk 1]
```diff
@@ -62,9 +62,8 @@ def __init__(self, filename, center_x, center_y, angle, speed, range, sound=None
 
         self.forward(self.speed)
 
-        # play the shot sound if present
-        if sound:
-            sound.play()
+    def destroy(self):
```
Comment (tobiasbp) [inline my_game.py:65]: We should not need this anymore, right? We can just use _kill()_ Was this not part of some workaround for the sound issue?

File: my_game.py [hunk 2]
```diff
@@ -270,7 +269,7 @@ def __int__(self, shot_list, target, level=1, **kwargs):
         arcade.schedule(self.change_dir, CONFIG['UFO_DIR_CHANGE_RATE'])
```
Comment (tobiasbp) [inline my_game.py:269]: It would be nice to not use _arcade.schedule()_ anymore. Can you make a timer based soulution?

File: my_game.py [hunk 3]
```diff
@@ -875,14 +874,24 @@ def on_update(self, delta_time):
         Movement and game logic
         """
 
+        #Ufo shooting
+        for ufo in self.ufo_list:
+            # Timer for when ufo should shoot
+            ufo.shoot_timer -= delta_time
```
Comment (tobiasbp) [inline my_game.py:880]: The countdown of the timer should be a part of the Ufo's _on_update(delta_time)_. Change to _on_update()_ if we currently use _update()_.

File: my_game.py [hunk 4]
```diff
@@ -899,24 +908,26 @@ def on_update(self, delta_time):
 
         # checks if ufo shot collides with player
         for ufo_shot_hit in self.player_sprite.collides_with_list(self.ufo_shot_list):
+            self.sound_explosion.play()
             self.player_sprite.lives -= 1
             self.player_sprite.reset()
             self.get_explosion(self.player_sprite.position)
-            ufo_shot_hit.kill()
+            ufo_shot_hit.destroy()
```
Comment (tobiasbp) [inline my_game.py:915]: What's wrong with arcade native _kill()_?

**Review by tobiasbp — CHANGES_REQUESTED** [review]
See comments on requested changes pls.

File: my_game.py (hunk 1 above)
Comment (TheRiddler4K) [reply my_game.py:65]: That is correct

File: my_game.py (hunk 2 above)
Comment (TheRiddler4K) [reply my_game.py:269]: Ok

File: my_game.py (hunk 3 above)
Comment (TheRiddler4K) [reply my_game.py:880]: Yes the problem was that ufo.update() didn't have delta time, but changing it to on_update fixes that?

**Review by tobiasbp — APPROVED** [review]
Nice update. Good commit messages. Thanks.

**Pull request merged**